settings

```diff
@@ -21,6 +21,18 @@ load_settings() {
   # Expected: FULLGPGKEY
   # Optional: FULLVERSION
   . "$RELEASEDIR/settings"
+
+  if [[ $PROJECT != foreman ]]; then
```

Comment:
Why do we need to do this before creating a dummy settings file?

Reply:
this is not before -- it's a function :)

Reply:
Right, my bad.